src/main/java/org/openpnp/vision/FluentCv.java

```diff
@@ -668,6 +668,8 @@ private FluentCv store(Mat mat, String... tag) {
 		if (tag != null && tag.length > 0) {
 			// Clone so that future writes to the pipeline Mat
 			// don't overwrite our stored one.
+			mat = stored.get(tag[0]);
+                        if(mat!=null) { mat.release(); }
```

Comment:
No. This.mat get the mat  value. normally this.mat should be released too,
but I'm not sure if that mat is used on some code
And because this I ignore it.
Mat have native pointer in it and this probably block normal memory GC or
something like that.